src/components/SignupForm.jsx

```diff
@@ -80,7 +80,7 @@ export default class SignupForm extends Component {
     return (
       <div className={styles.signup}>
         <div>
-          <H1>Build borderless commericial systems on the blockchain.</H1>
+          <H1>Build borderless commericial systems on the blockchain</H1>
```

Comment:
caught typo in commercial here

Reply:
Fixed now. Thanks! The more eyes the better on things like this!